doc/manual/risk.rst

```diff
@@ -446,6 +446,185 @@ For further information regarding the theoretical background of the
+3. Creation of original network from the exposure model
+4. Update of the functionality of each component to the originally created
+   infrastructure network and removal of the non-functional/non-operational
+   components
+5. Network analysis based on the original and updated infrastructure network
+6. Computation of the following performance metrics, based on connectivity
+   analysis:
+
+  * Complete Connectivity Loss (CCL)
+  * Partial Connectivity Loss (PCL)
+  * Weighted Connectivity Loss (WCL)
+  * Efficiency Loss (EL)
+
+In summary, for the computation of infrastructure risk at connectivity level in
+OpenQuake, the input requirements and the outputs obtained are shown
+in :numref:`fig:infrastructure-io`.
+
+.. figure:: figures/risk/infrastructure-io.png
+   :alt: Input models required, and output obtained in OpenQuake for infrastructure risk assessment at connectivity level
+   :name: fig:infrastructure-io
+   :width: 95%
+
+   Input models required, and output obtained in OpenQuake for infrastructure risk assessment at connectivity level
+
+Specification of the Outputs Obtained
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* For the classic generic case where "demand" and "supply" are explicity
+  mentioned in the column **demand_or_supply**, all four metrics are computed
+  for overall network. Also, at nodal level, all the four metrics are
+  calculated.
+
+* If some nodes behave as both supply and demand, and are assigned as "both"
+  or "TAZ" (traffic analysis zone) in the column **demand_or_supply**, PCL,
+  WCL and EL are computed for both nodal level and overall network.
+
+* If there is no assignment of "demand" or "supply" to the column
+  **demand_or_supply**, only EL is computed for overall network. Also, at
+  nodal level, for all the nodes, EL only is computed.
+
+For the simplification, :numref:`fig:infrastructure-output-by-nodes-func` has been added to understand what
+can be obtained by various specifications.
+
+.. figure:: figures/risk/infrastructure-output-by-nodes-func.png
+   :alt: Output computed from the implementation according to the specification of the function of the nodes
+   :name: fig:infrastructure-output-by-nodes-func
+   :width: 95%
+
+   Output computed from the implementation according to the specification of the function of the nodes
+
+Mainly, the library, NetworkX (Aric et al. 2008) has been used during the
+implementation. Further details can be found in Poudel et al. 2023, and Poudel
+and Pitilakis 2023. Also, much concept during the implementation has been drawn
+from Pitilakis et al. 2014.
+
+References
+~~~~~~~~~~
+
+1. Aric A. Hagberg, Daniel A. Schult and Pieter J. Swart (2008) Exploring
+   network structure, dynamics, and function using NetworkX, in Proceedings of
+   the 7th Python in Science Conference (SciPy2008), Gäel Varoquaux, Travis
+   Vaught, and Jarrod Millman (Eds), (Pasadena, CA USA), pp. 11–15
+2. Poudel A, Pitilakis K, Silva V, Rao A (2023) Infrastructure Seismic Risk
```

Comment:
Change to :
Poudel, A., Pitilakis, K., Silva, V. and Rao, A., 2023. Infrastructure seismic risk assessment: an overview and integration to contemporary open tool towards global usage. Bulletin of Earthquake Engineering. https://doi.org/10.1007/s10518-023-01693-z

Reply:
Done